### ISSUE TITLE
Appropriate use of object properties

### ISSUE THREAD

**Issue by Melek-C** [h3abionet/afpo#16]
Two upper level classes, Continent and Population.

TODO Definition formatting: should follow standard formatting. Beginning with the parent term. The definition sentence structure should be grammatically correct.

This ontology is using the **property "see Also"** for sources of the Class data/information. 
expect to see this kind of information under 'definition_source'
TODO -- Consult IAO (Information Artifact Ontology) 

There are many AfPO specific properties, e.g., AfPO:0000152, which is annotated
with a PubMed URL. Instead of the AfPO property, the ontology could use hasDbXref
TODO. - review duplications in property annotations:

**Comment by anitacaron** [comment]
- [ ] Split synonym list in many exact synonym annotations

**Comment by anitacaron** [comment]
> TODO Definition formatting: should follow standard formatting. Beginning with the parent term. The definition sentence structure should be grammatically correct.

> This ontology is using the property "see Also" for sources of the Class data/information.
expect to see this kind of information under 'definition_source'
TODO -- Consult IAO (Information Artifact Ontology)

Fixed in PR #5

**Comment by anitacaron** [comment]
> There are many AfPO specific properties, e.g., AfPO:0000152, which is annotated
with a PubMed URL. Instead of the AfPO property, the ontology could use hasDbXref
TODO. - review duplications in property annotations:

The AfPO-specific annotations bring more information; for example, there is an annotation for cross reference for population (AfPO:0000600) and another for language (AfPO:0000601). We classified these two annotations under oboInOwl:hasDbXref. For population:genetic:study (AfPO:0000152) and population:microbiome:study (AfPO:0000226) we classified them as AfPO:0000600.  Done in PR #41
